src/cmdlib.sh

```diff
@@ -832,4 +884,4 @@ builddir = builds.get_build_dir('${buildid}')
 buildmeta = builds.get_build_meta('${buildid}')
 cmdlib.import_ostree_commit('${workdir:-$(pwd)}/tmp/repo', builddir, buildmeta)
 ")
-}
+}
```

Comment:
Minor: looks like we're eating the file's final newline here.

Reply:
I think there was an extra space after` } `which is not needed.